wishes/views.py

```diff
@@ -0,0 +1,64 @@
+import json, requests
+
+from django.http      import JsonResponse
+from django.views     import View
+from django.db.utils  import IntegrityError
+
+from .models         import Wish
+from products.models import Food
+from users.utils     import login_decorator
+
+class WishView(View):
+    @login_decorator
+    def post(self, request):
+        try:
+            data = json.loads(request.body)
+
+            if Wish.objects.filter(user=request.users, food_id=data['food_id']).exists():
+                return JsonResponse({'message':'DUPLICATED_FOOD'}, status=400)
+
+            wish = Wish(user=request.users,food_id=data['food_id'])
+            wish.save()
+        
+            return JsonResponse({'message':'SUCCESS'}, status=201)
+        
+        except Food.DoesNotExist:
+            return JsonResponse({'message':'NOT EXIST'}, status=404)
+        except Food.MultipleObjectsReturned:
+            return JsonResponse({'message':'MULTIPLE EXIST'}, status=400)
```

Comment:
해당 exception은 위 코드에서는 발생하지 않을 것 같습니다.

```suggestion
        except Food.DoesNotExist:
            return JsonResponse({'message':'NOT EXIST'}, status=404)
        except Food.MultipleObjectsReturned:
            return JsonResponse({'message':'MULTIPLE EXIST'}, status=400)
```

Reply:
수정하였습니다.